api/data/refactoring.py

```diff
@@ -154,3 +158,118 @@ def save_lovd_as_vcf(data, save_to="./lovd.vcf"):
 
             f.write("\t".join(record))
             f.write("\n")
+
+
+def get_variant_ids_from_clinvar_name_api(name: str, count: int = 100):
+    """
+    Extracts variant ids from ClinVar `name` variable. /n
+    key of dictionary is the size of the list of ids.
+
+    :param str name: name of variant
+    :param int count: number of ids to extract
+    :returns: ids of variants
+    :rtype: str
+    """
+
+    result = {}
```

Comment:
Keep variables closer to their first usage. For this variable, you can initiate it with required items inside instead of assigning them.